src/components/FormComponents/EditorController/EditorController.tsx

```diff
@@ -1,28 +1,115 @@
+import React, { useCallback, useRef } from 'react';
 import { Controller, FieldValues } from 'react-hook-form';
+import { ExposeParam, InsertContentGenerator } from 'md-editor-rt';
+import { useTranslation } from 'react-i18next';
+import 'md-editor-rt/lib/style.css';
 
-import { StyledEditor } from './EditorController.styles';
+import { MarkExtension } from 'components/MarkDownEditor/extensions/MarkExtension';
+import { TrashExtension } from 'components/MarkDownEditor/extensions/TrashExtension';
+import { AlignTextExtension } from 'components/MarkDownEditor/extensions/AlignTextExtension';
+import { AudioUploadExtension } from 'components/MarkDownEditor/extensions/AudioUploadExtension';
+import { ImageUploadExtension } from 'components/MarkDownEditor/extensions/ImageUploadExtension';
+import { VideoUploadExtension } from 'components/MarkDownEditor/extensions/VideoUploadExtension';
+import { LANGUAGE_BY_DEFAULT } from 'components/MarkDownEditor';
```

Comment:
Maybe we can add index file to import all extensions from one place?

Reply:
done